metadata-ingestion/src/datahub/ingestion/source/sql/athena.py

```diff
@@ -105,7 +105,10 @@ def gen_database_containers(
 
     def gen_schema_key(self, db_name: str, schema: str) -> DatabaseKey:
         return DatabaseKey(
-            platform=self.platform, instance=self.config.env, database=schema
+            platform=self.platform,
+            environment=self.config.env,
+            instance=self.config.platform_instance,
```

Comment:
I thought if instance was not provided, we were going to fallback to env... is this not the case anymore?